log-data/src/main/java/io/streamzi/cloudeventflow/log/data/DataLogger.java

```diff
@@ -5,7 +5,7 @@
  */
 package io.streamzi.cloudeventflow.log.data;
 
-import io.cloudevents.CloudEvent;
+import io.streamzi.cloudevents.CloudEvent;
```

Comment:
I think this was done w/ my PR, and needs to be rebased

Reply:
Done...